src/app.js

```diff
@@ -25,107 +25,92 @@ export function app(props) {
     }
   }
 
-  function render(cb) {
-    element = patch(
-      appRoot,
-      element,
-      node,
-      (node = appView(appState, appActions)),
-      (skipRender = !skipRender)
+  function render() {
+    flush(
+      (element = patch(
+        appRoot,
+        element,
+        node,
+        (node = appView(appState, appActions)),
+        (skipRender = !skipRender)
+      ))
     )
-    while ((cb = lifecycleCallbackStack.pop())) cb()
   }
 
-  function hydrate(element, map) {
-    return (
-      element &&
-      h(
-        element.tagName.toLowerCase(),
-        {},
-        map.call(element.childNodes, function(element) {
-          return element.nodeType === 3
-            ? element.nodeValue
-            : hydrate(element, map)
-        })
-      )
-    )
+  function flush(cb) {
```

Comment:
When does `flush` actually receive a `cb`?

Reply:
To avoid declaring the variable inside the function with `var`. Just a silly technique to save a few bytes here.